tools/api-markdown-documenter/README.md

```diff
@@ -11,6 +11,19 @@ It is similar to [API-Documenter][] and is heavily based upon it and uses it und
 **Note**: this library does not currently offer a Command Line Interface (CLI).
 One may be added in the future, but for now this library is intended to be consumed programmatically.
 
+## Glossary
+
+The following terms are leveraged heavily in this package's APIs and documentation.
+
+-   **API Model**: Refers to a complete API suite, comprised of one or more packages.
+    This often corresponds to all of the packages in the mono-repo, or a series of packages that are published together.
+    It is generally represented via the [ApiModel][] type, from the [@microsoft/api-extractor-model](https://github.com/microsoft/rushstack/tree/main/libraries/api-extractor-model) library.
+    -   In some places, this library refers to an `API Model` in terms of a directory or directory path.
+        In these cases, it is referring to a directory that contains the set of `.api.json` files (generated per-package by `API-Extractor`).
+-   **API Item**: Refers to a single item exported by one or more packages in the `API Model`.
```

Comment:
I read through the linked documentation, and it doesn't do a great job explaining what API Item actually is (it just says what types it can be). Are API Items always derived from doc comments? Are there other cases where API Extractor creates them?

Reply:
Okay, I'll elaborate here.

Reply:
@TommyBrosman I took a stab at an update that includes some examples. Let me know if that helps clarify things, or if more is needed.